cirq/ops/pauli_string.py

```diff
@@ -698,7 +698,10 @@ def zip_paulis(
     ) -> Iterator[Tuple[pauli_gates.Pauli, pauli_gates.Pauli]]:
         return (paulis for qubit, paulis in self.zip_items(other))
 
-    def _commutes_(self, other: Any, atol: float) -> Union[bool, NotImplementedType, None]:
+    def _commutes_(
```

Comment:
this seems unrelated - can you revert this formatting change?

Reply:
in fact there seems to be some inconsistency around this in the codebase:

![image](https://user-images.githubusercontent.com/635073/105259766-42ffe300-5b41-11eb-9c96-211d02c77d87.png)

Alright, let's keep the * in front of atol, I'll open an issue to cleanup this inconsistency.